app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -28,7 +28,7 @@ def index
       private
 
       def request_params
-        params.permit(:name, :description, :requester, :content => {})
+        params.permit(:name, :description, :owner, :content => {})
```

Comment:
We can drop `owner` from the api and remove it from `openapi.json` too. There was a Jira ticket for it too.